cypress/integration/js2048Game.spec.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+'use strict';
+
+Cypress.Commands.add('shuffleBoxes', (arrow1, arrow2, times) => {
+  cy.get('.button.start').click();
+
+  for (let n = 0; n < times; n++) {
+    cy.get('body').type(arrow1);
+    cy.get('body').type(arrow2);
+  }
+  cy.get('.game-score').invoke('text').then(parseFloat).should('be.gt', 1);
```

Comment:
if you shuffle 1 time, the score is unlikely to be >1. So how about we move this check outside of shuffleBoxes method?

Reply:
remove this check

Reply:
done